buildspec.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.10    
+    commands:
+      - pip install --upgrade pip
+      - pip install -r source/lambda/requirements.txt -t source/lambda/
```

Comment:
I don't like putting things like this in the buildspec because it makes the local and pipeline builds different. I'd rather put it in build-s3-dist if it's really necessary, and clean up afterward.